core/tests/test_utils.py

```diff
@@ -1537,6 +1538,53 @@ def swap(self, obj, attr, newvalue):
         finally:
             setattr(obj, attr, original)
 
+    @contextlib.contextmanager
```

Comment:
Use set_config_property() instead

Reply:
Done.

Reply:
Oh cool, didn't know that function existed! :)

Thanks for the suggestion!